lib/prometheus/middleware/exporter.rb

```diff
@@ -0,0 +1,89 @@
+# encoding: UTF-8
+
+require 'prometheus/client'
+require 'prometheus/client/formats/text'
+
+module Prometheus
+  module Middleware
+    # Exporter is a Rack middleware that provides a sample implementation of
+    # a Prometheus HTTP client API.
```

Comment:
I'm pretty sure this isn't implementing a Prometheus HTTP client API. It's exposition.